relayer/src/config.rs

```diff
@@ -6,6 +6,8 @@ pub struct Config {
     pub max_fork_len: u64,
     pub sleep_time_on_fail_sec: u64,
     pub sleep_time_on_reach_last_block_sec: u64,
+    pub sleep_time_after_iteration_sec: u64,
```

Comment:
Perhaps more descriptive name? Iteration is very vague.
`sleep_time_after_sync_sec`?

Reply:
fixed: https://github.com/Near-One/btc-light-client-contract/pull/51/commits/a39da890e520433aa74828aaf56cd9c6a88b2a3a